components/purchase_modal/purchase_modal.tsx

```diff
@@ -203,6 +221,78 @@ function Card(props: CardProps) {
     );
 }
 
+function DelinquencyCard(props: DelinquencyCardProps) {
+    const seeHowBillingWorks = (
+        e: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    ) => {
+        e.preventDefault();
+        trackEvent(
+            TELEMETRY_CATEGORIES.CLOUD_ADMIN,
+            'click_see_how_billing_works',
+        );
+        window.open(CloudLinks.BILLING_DOCS, '_blank');
```

Comment:
@johndavidlugtu do we need to update the billing docs to include this new delinquency flow?

Reply:
Yup - already on @justinegeffen 's radar

Reply:
In the meantime, the link can be pointed here: https://docs.mattermost.com/about/cloud-subscriptions.html#failed-or-late-payments

Reply:
I've updated it to point to this in the delinquency card